ngi_reports/utils/statusdb.py

```diff
@@ -50,9 +66,15 @@ def get_entry(self, name, use_id_view=False):
             if self.log:
                 self.log.warn("no entry '{}' in {}".format(name, self.db))
             return None
-        return self.db.get(view.get(name))
+        if type(self) is NanoporeRunConnection:
+            doc = self.cloudant.get_document(db=self.dbname, doc_id=view.get(name)).get_result()
+        else:
+           doc = self.db.get(view.get(name))
```

Comment:
Is there a difference in indentation between the doc line here?

Reply:
Copypasta 🍝